resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -514,10 +514,21 @@
          if annotation.annotation|type|subtype('data:annotation:gtf')
          else 'gff3' %}
 
+      # Remove empty gene_id entries from annotation file if source is UCSC
```

Comment:
I think the offending lines get modified `gene_id ""` -> `gene_id "unknown"`, not removed. Please update the comment. Otherwise looks good.